Mathlib/LinearAlgebra/Matrix/SchurComplement.lean

```diff
@@ -444,14 +444,20 @@ theorem det_one_sub_mul_comm (A : Matrix m n α) (B : Matrix n m α) :
   rw [sub_eq_add_neg, ← Matrix.neg_mul, det_one_add_mul_comm, Matrix.mul_neg, ← sub_eq_add_neg]
 #align matrix.det_one_sub_mul_comm Matrix.det_one_sub_mul_comm
 
-/-- A special case of the **Matrix determinant lemma** for when `A = I`.
-
-TODO: show this more generally. -/
+/-- A special case of the **Matrix determinant lemma** for when `A = I`. -/
 theorem det_one_add_col_mul_row (u v : m → α) : det (1 + col u * row v) = 1 + v ⬝ᵥ u := by
   rw [det_one_add_mul_comm, det_unique, Pi.add_apply, Pi.add_apply, Matrix.one_apply_eq,
     Matrix.row_mul_col_apply]
 #align matrix.det_one_add_col_mul_row Matrix.det_one_add_col_mul_row
 
+/-- The **Matrix determinant lemma** -/
+theorem matrix_determinant_lemma (A: Matrix m m α) (U: Matrix m n α)
+    (V: Matrix n m α) (hA: IsUnit A.det):
```

Comment:
```suggestion
theorem matrix_determinant_lemma (A : Matrix m m α) (U : Matrix m n α)
    (V : Matrix n m α) (hA : IsUnit A.det) :
```
Perhaps the argument `A` could be made implicit, since it is mentioned already in `hA`

Reply:
> Perhaps the argument `A` could be made implicit, since it is mentioned already in `hA`

Sure, please let me know how to do it

Reply:
sure, see the other suggestion